From b128644b9641d004fb7d7e6b4e0e5ad04845ba74 Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
Date: Tue, 16 Jul 2013 02:11:11 +0200
Subject: [PATCH 631/675] sh-pfc: sh7734: Use the common GP port style macros

The SoC has a bank-style PFC. Replace the custom-defined macros with
common macros from sh-pfc.h.

Signed-off-by: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
---
 drivers/pinctrl/sh-pfc/sh_pfc.h | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/drivers/pinctrl/sh-pfc/sh_pfc.h b/drivers/pinctrl/sh-pfc/sh_pfc.h
index 9fef062..4220620b 100644
--- a/drivers/pinctrl/sh-pfc/sh_pfc.h
+++ b/drivers/pinctrl/sh-pfc/sh_pfc.h
@@ -237,23 +237,6 @@ struct sh_pfc_soc_info {
 	PORT_1(fn, pfx##6, sfx), PORT_1(fn, pfx##7, sfx),		\
 	PORT_1(fn, pfx##8, sfx), PORT_1(fn, pfx##9, sfx)
 
-#define PORT_10_REV(fn, pfx, sfx)					\
-	PORT_1(fn, pfx##9, sfx), PORT_1(fn, pfx##8, sfx),		\
-	PORT_1(fn, pfx##7, sfx), PORT_1(fn, pfx##6, sfx),		\
-	PORT_1(fn, pfx##5, sfx), PORT_1(fn, pfx##4, sfx),		\
-	PORT_1(fn, pfx##3, sfx), PORT_1(fn, pfx##2, sfx),		\
-	PORT_1(fn, pfx##1, sfx), PORT_1(fn, pfx##0, sfx)
-
-#define PORT_32(fn, pfx, sfx)						\
-	PORT_10(fn, pfx, sfx), PORT_10(fn, pfx##1, sfx),		\
-	PORT_10(fn, pfx##2, sfx), PORT_1(fn, pfx##30, sfx),		\
-	PORT_1(fn, pfx##31, sfx)
-
-#define PORT_32_REV(fn, pfx, sfx)					\
-	PORT_1(fn, pfx##31, sfx), PORT_1(fn, pfx##30, sfx),		\
-	PORT_10_REV(fn, pfx##2, sfx), PORT_10_REV(fn, pfx##1, sfx),	\
-	PORT_10_REV(fn, pfx, sfx)
-
 #define PORT_90(fn, pfx, sfx) \
 	PORT_10(fn, pfx##1, sfx), PORT_10(fn, pfx##2, sfx),		\
 	PORT_10(fn, pfx##3, sfx), PORT_10(fn, pfx##4, sfx),		\
-- 
1.8.2

